source/Halibut.Tests/Timeouts/SendingAndReceivingRequestMessagesTimeoutsFixture.cs

```diff
@@ -141,16 +141,25 @@ public async Task WhenThenNetworkIsPaused_WhileSendingARequestMessage_ATcpWriteT
                 await echo.SayHelloAsync("Make a request to make sure the connection is running, and ready. Lets not measure SSL setup cost.");
 
                 var echoServiceTheErrorWillHappenOn = clientAndService.CreateClient<IEchoService, IAsyncClientEchoService>(IncreasePollingQueueTimeout());
-
-                var stringToSend = Some.RandomAsciiStringOfLength(numberOfBytesBeforePausingAStream * 20);
+                
                 var sw = Stopwatch.StartNew();
-                var e = (await AssertAsync.Throws<HalibutClientException>(() => echoServiceTheErrorWillHappenOn.SayHelloAsync(stringToSend))).And;
+                var e = (await AssertAsync.Throws<HalibutClientException>(() =>
+                {
+                    var stringToSend = Some.RandomAsciiStringOfLength(numberOfBytesBeforePausingAStream * 20);
```

Comment:
It was driving me nuts debugging in this area, and having VS take ages to "evaluate local variables" because of this large string. 

So I made sure it was no longer in scope. Solved!